draft-ietf-webpush-encryption.md

```diff
@@ -42,17 +31,21 @@ normative:
       - org: National Institute of Standards and Technology (NIST)
     date: July 2013
     seriesinfo: NIST PUB 186-4
-  X.692:
+  X9.62:
      title: "Public Key Cryptography For The Financial Services Industry: The Elliptic Curve Digital Signature Algorithm (ECDSA)"
      author:
        - org: ANSI
      date: 1998
      seriesinfo: ANSI X9.62
+  FIPS180-4:
+    title: NIST FIPS 180-4, Secure Hash Standard
+    author:
+      name: NIST
+      ins: National Institute of Standards and Technology, U.S. Department of Commerce
+    date: 2012-03
+    target: http://csrc.nist.gov/publications/fips/fips180-4/fips-180-4.pdf
```

Comment:
Document at link notes that it has been superseded by http://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.180-4.pdf.